Common/Inc/zp_assert.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
Pragma once is the same as the include guards directly below. We use include guards throughout our project so I would stick to that
```suggestion
```